triage_process.md

```diff
@@ -39,16 +39,32 @@ Stale pull requests|[Core/Provider stale pull requests](https://github.com/pulls
+[Core/Provider stale pull requests]:                        https://github.com/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+label%3Astale+archived%3Afalse+user%3AManageIQ+
+[UI/API unassigned issues]:                                 https://github.com/issues?utf8=%E2%9C%93&q=is%3Aissue+-label%3A%22help+wanted%22+repo%3AManageiq%2Fmanageiq-ui-classic+repo%3AManageiq%2Fmanageiq-api+repo%3AManageIQ%2Fui-components+repo%3AManageIQ%2Freact-ui-components+repo%3AManageiq%2Fmanageiq-ui-service+is%3Aopen+no%3Aassignee+
+[UI/API issues without scope label]:                        https://github.com/issues?page=1&q=is%3Aissue+-label%3A%22help+wanted%22+-label%3Arefactoring+-label%3A%22technical+debt%22+-label%3Abug+-label%3Aenhancement+repo%3AManageiq%2Fmanageiq-ui-classic+repo%3AManageiq%2Fmanageiq-api+repo%3AManageIQ%2Fui-components+repo%3AManageIQ%2Freact-ui-components+repo%3AManageiq%2Fmanageiq-ui-service+is%3Aopen&utf8=%E2%9C%93
+[UI/API stale issues]:                                      https://github.com/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3Astale+repo%3AManageiq%2Fmanageiq-ui-classic+repo%3AManageiq%2Fmanageiq-api+repo%3AManageIQ%2Fui-components+repo%3AManageIQ%2Freact-ui-components+repo%3AManageiq%2Fmanageiq-ui-service+is%3Aopen+
+[UI/API unassigned pull requests]:                          https://github.com/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+-label%3A%22help+wanted%22+repo%3AManageiq%2Fmanageiq-ui-classic+repo%3AManageiq%2Fmanageiq-api+repo%3AManageIQ%2Fui-components+repo%3AManageIQ%2Freact-ui-components+repo%3AManageiq%2Fmanageiq-ui-service+is%3Aopen+no%3Aassignee+
+[UI/API assigned pull requests without scope label]:        https://github.com/issues?page=2&q=is%3Apr+-label%3A%22help+wanted%22+-label%3Arefactoring+-label%3A%22technical+debt%22+-label%3Abug+-label%3Aenhancement+repo%3AManageiq%2Fmanageiq-ui-classic+repo%3AManageiq%2Fmanageiq-api+repo%3AManageIQ%2Fui-components+repo%3AManageIQ%2Freact-ui-components+repo%3AManageiq%2Fmanageiq-ui-service+is%3Aopen&utf8=%E2%9C%93
+[UI/API stale pull requests]:                               https://github.com/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+label%3Astale+archived%3Afalse+user%3AManageIQ+
```

Comment:
Worth noting, but I was unsure if this was intentional:

This link is exactly the same as `Core/Provider stale pull requests`, so not sure if that was by design (or there is no better way to filter in GH) or if that was a mistake.